package.json

```diff
@@ -35,54 +35,53 @@
     "release:github": "npx conventional-github-releaser -p angular"
   },
   "devDependencies": {
-    "@types/jest": "29.5.12",
+    "@types/jest": "30.0.0",
     "@types/js-yaml": "4.0.9",
     "@types/json-schema": "7.0.15",
-    "@types/json-stable-stringify": "1.0.36",
-    "@types/lodash": "4.17.7",
-    "@types/node": "22.2.0",
-    "@types/react": "18.3.3",
-    "@types/yargs": "17.0.33",
-    "@typescript-eslint/eslint-plugin": "8.1.0",
-    "@typescript-eslint/parser": "8.1.0",
+    "@types/lodash": "4.17.21",
+    "@types/node": "24.10.1",
+    "@types/react": "19.2.7",
+    "@types/yargs": "17.0.35",
+    "@typescript-eslint/eslint-plugin": "8.48.1",
+    "@typescript-eslint/parser": "8.48.1",
     "conventional-changelog-cli": "^5.0.0",
     "copyfiles": "2.4.1",
-    "eslint": "9.9.0",
-    "eslint-config-prettier": "9.1.0",
-    "eslint-plugin-prettier": "5.2.1",
-    "eslint-plugin-react": "7.35.0",
-    "husky": "9.1.4",
-    "jest": "29.7.0",
+    "eslint": "9.39.1",
+    "eslint-config-prettier": "10.1.8",
+    "eslint-plugin-prettier": "5.5.4",
+    "eslint-plugin-react": "7.37.5",
+    "husky": "9.1.7",
+    "jest": "30.2.0",
     "jest-expect-message": "1.1.3",
-    "lint-staged": "15.2.9",
-    "ts-jest": "29.2.4",
+    "lint-staged": "16.2.7",
+    "ts-jest": "29.4.6",
     "ts-node": "10.9.2",
     "tsconfig-paths": "4.2.0",
-    "tsx": "4.17.0"
+    "tsx": "4.21.0"
   },
   "dependencies": {
     "figures": "6.1.0",
-    "fuse.js": "7.0.0",
-    "got": "14.4.2",
+    "fuse.js": "7.1.0",
+    "got": "14.6.5",
     "handlebars": "4.7.8",
-    "ink": "5.0.1",
-    "ink-link": "4.1.0",
-    "ink-select-input": "6.0.0",
+    "ink": "6.5.1",
+    "ink-link": "5.0.0",
+    "ink-select-input": "6.2.0",
     "ink-spinner": "5.0.0",
     "ink-text-input": "6.0.0",
-    "joi": "17.13.3",
-    "js-yaml": "4.1.0",
+    "joi": "18.0.2",
+    "js-yaml": "4.1.1",
     "json-schema": "0.4.0",
-    "json-stable-stringify": "1.1.1",
+    "json-stable-stringify": "1.3.0",
     "latest-version": "9.0.0",
     "lodash": "4.17.21",
     "node-machine-id": "1.1.12",
     "object-assign": "4.1.1",
-    "prettier": "3.4.2",
-    "react": "18.3.1",
-    "sort-keys": "5.0.0",
-    "typescript": "5.5.4",
-    "yargs": "17.7.2"
+    "prettier": "3.7.3",
+    "react": "19.2.0",
```

Comment:
React team released a [security advisory](https://react.dev/blog/2025/12/03/critical-security-vulnerability-in-react-server-components) around a critical vulnerability in this version of react. Can we please upgrade it to [19.2.1](https://github.com/facebook/react/releases/tag/v19.2.1) before  merging it?
CC: @akashrpo

Reply:
We can close this PR for now, as we’ll track the upgrade to `pnpm` request as a separate task